dashboard/test/models/regional_partners_school_districts_test.rb

```diff
@@ -25,4 +25,5 @@ class RegionalPartnersSchoolDistrictTest < ActiveSupport::TestCase
     assert_equal partner.name, 'Center for STEM Education, The University of Texas at Austin'
     assert_equal partner.group, 2
   end
+  # rubocop:enable Style/NumericLiterals
```

Comment:
If this is just for a single line, we could instead disable that line [directly](http://rubocop.readthedocs.io/en/latest/configuration/#disabling-cops-within-source-code) via:

```rb
district = SchoolDistrict.find(4800014) # rubocop:disable Style/NumericLiterals
```

Reply:
Though we could, I think it is more readable (with little downside) to disable the cop for the method rather than individually for three lines. If you disagree, I'm happy to do as you suggest.

Reply:
OIC. The default diff view was only showing one of the lines. I agree, with 3 we might as well disable for the whole method 👍

Reply:
Note that, on closer inspection, the disable is only needed for two based on our [settings](https://github.com/code-dot-org/code-dot-org/blob/staging/.rubocop.yml#L43-L44).